### DISCUSSION TITLE
質問のコメントにCodePen埋め込みコンテンツが表示されない

### DISCUSSION THREAD

**Discussion by blue32a** [increments/qiita-discussions#520] [要望・アイデア・不具合]
いつもお世話になっております。

質問のコメントにCodePenの埋め込みコンテンツを使用したのですが、コメント投稿後はコンテンツが表示されない現象があったので報告いたします。

入力時のプレビューでは表示されます。
恐らくですが初期状態で折りたたまれるコメントだと表示されないようです。折りたたまれない最初のコメントでは表示されました。

この現象が確認できる質問はこちらです。
https://qiita.com/jankenpon0205/questions/1d01d17583cda843d832

私の回答から「残りx件のコメントを表示」を開いてください。
埋め込みコンテンツが次のように表示されています。

> See the Pen Qiita Q&A 1d01d17583cda843d832 by blue32a (@blue32a) on CodePen.

なおCodePen以外は検証できておりません。
ご確認いただけますと幸いです。

**Comment by wataru86** [comment]
@blue32a

ご不便をおかけして申し訳ありません🙏
現在、セキュリティの観点から回答に対するコメントでは CodePen が使用できないようになっています。
回答コメントでも CodePen が使えるようにするかどうかや、プレビューとの違いについて検討したいと思います🙇

**Reply by blue32a** [reply]
回答ありがとうございます。
なるほど、セキュリティの問題があるのですね。
引き続き検討お願いいたします🙇

**Reply by wataru86** [reply]
@blue32a

お待たせしました！
検討の結果、問題ないと判断し、回答に対するコメントでも CodePen を使えるようにしました！
https://qiita.com/release-notes/2024-05-27

**Reply by blue32a** [reply]
ありがとうございます！
